src/js/actions/ActionsCreators.js

```diff
@@ -280,8 +279,11 @@ export function setPayout(payload) {
     if (typeof payout === 'undefined') {
       payout = getState().getIn(['storage', 'payout']);
     }
+    else if (isNaN(payout*1) || !Number.isInteger(payout*1) || parseInt(payout) < 0 || parseInt(payout) > 100) {
```

Comment:
@qingweibinary I mean putting the `+` in line 280 won't help the line 282, because it won't go into line 280 to become `10.1` because it won't match the first condition as it isn't undefined~